cmd/opm/alpha/veneer/semver.go

```diff
@@ -37,7 +37,7 @@ func newSemverCmd() *cobra.Command {
 			case "yaml":
 				write = declcfg.WriteYAML
 			case "mermaid":
-				write = semver.MermaidChannelWriter
+				write = declcfg.WriteMermaidChannels
```

Comment:
Non-blocking: Hmm, a write function that takes a config variable to determine output format might be nice.